course_os.cbp

```diff
@@ -0,0 +1,269 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This together with all(?) other CMakeFiles likely needs to go.

Reply:
fixed